src/components/overview/ProductInformation/AverageRating.jsx

```diff
@@ -12,7 +12,7 @@ const AverageRating = ({state, updateState}) => {
 
   const scrollToReview = () => {
     window.scrollTo({
-      top: 1500,
+      top: 3000,
```

Comment:
Could you not scroll to the review div here? Scrolling a fixed distance seems potentially unreliable.

Reply:
good point! I will change it to scroll to the specific ID instead of fixed distance as the below. Can I create ID as reviews at your reviews component?

- document.getElementById('reviews').scrollIntoView(true);

Reply:
That would work, and you can also do:
`document.getElementsByClass('reviews')[0].scrollIntoView(true);`

Reply:
I updated using the id! because the code that you suggest somehow does not work for me..